dev-tools/packaging/package_test.go

```diff
@@ -183,6 +184,21 @@ func checkZip(t *testing.T, file string) {
 	checkLicensesPresent(t, "", p)
 }
 
+var npcapConfigPattern = regexp.MustCompile("Windows Npcap installation settings")
+
+func checkNpcapNotices(pkg, file string, contents io.Reader) error {
+	if !strings.Contains(pkg, "packetbeat") || !strings.Contains(pkg, "windows") {
+		return nil
+	}
+	switch file {
+	case "packetbeat.yml", "packetbeat.reference.yml":
+		if !npcapConfigPattern.MatchReader(bufio.NewReader(contents)) {
+			return fmt.Errorf("Npcap config section not found in config file %s in %s", file, pkg)
```

Comment:
I think it would be nice to test that it does not exist in OSS packages or in non-Windows packages. The latter would require inspecting non-zip files though.

Reply:
Yes. The design of the inspector is intended to allow that if we want it.

Reply:
The reason I didn't is that the committed linux version is essentially a canary for that.